themes/src/main/resources/theme/base/login/register.ftl

```diff
@@ -104,6 +104,17 @@
                             </span>
                         </#if>
                     </div>
+                    <div class="${properties.kcInputWrapperClass!}" id="password-entropy" data-password-input="password">
+                        <div class="progress">
+                            <div class="progress-bar" role="status" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100"
```

Comment:
The `status` role doesn't understand the `value*` attributes, so they can be removed. As long as there is a human-readable text that is inside of the element it will be legible to screen readers.
```suggestion
                            <div class="progress-bar" role="status"
```